mypyc/emitmodule.py

```diff
@@ -73,17 +131,32 @@ def compile_modules_to_c(result: BuildResult, module_names: List[str],
             for fn in module.functions:
                 ops.extend(format_func(fn))
                 ops.append('')
-    # Generate C code.
+
     source_paths = {module_name: result.files[module_name].path
                     for module_name in module_names}
-    generator = ModuleGenerator(literals, modules, source_paths, shared_lib_name,
-                                compiler_options.multi_file)
-    return generator.generate_c_for_modules()
+
+    names = NameGenerator([[source.module for source in sources] for sources, _ in groups])
+
+    # Generate C code for each compilation group. Each group will be
+    # compiled into a separate extension module, so we produce
+    # a separate group of C source for each of them.
```

Comment:
"Group of C source" is not entirely clear. Do we generate a separate file for each file in a group?